docs/USAGE-GUIDE.md

```diff
@@ -77,6 +77,31 @@ And when invoking CMake for your project, do one of two things:
  1. Set the `CMAKE_INSTALL_PREFIX` variable with the path to your s2n-tls build.
  2. If you have globally installed s2n-tls, do nothing, it will automatically be found.
 
+
+### 32 Bit Platforms
+s2n-tls has support for 32 bit builds, but they are not built or tested in our CI, so they may or may not work. If you run into any issue let us know! We welcome pull requests and/or suggestions about how to better support 32 bit customers!
+
```

Comment:
I'm not sure about this phrasing. What support do we actually have if we haven't built or tested? And aren't you adding testing right now? Previously, we haven't been accepting 32-bit related pull requests specifically because we have no tests.

Reply:
This phrasing was largely pulled from the Rust team's description of [tier 3 platform support](https://doc.rust-lang.org/beta/rustc/platform-support.html#tier-3). The "support" is currently just limited to the fact that this toolchain now exists which makes it easy to do a 32 bit cross compile, and we also did take action to unblock a failing build on 32 bit platforms.

I agree that it's certainly not a comfortable level of support but I think that `they may or may not work` calls that out relatively clearly.

If you want we could move to more of a warning
```
s2n-tls does not support 32 bit platforms. 32 bit platform builds and tests are not run as part of CI. Customers use s2n-tls in 32 bit platforms at their own risks. PRs related to 32 bit-only issues will not be accepted.
```
but that tone feels a bit harsh, given that we already have customers using 32 bit builds.

> And aren't you adding testing right now? 

In process, yes, but it isn't actually running in our CI. Once we have it running in our CI we can edit this comment appropriately.

Reply:
I think you can probably find a middle ground and do the warning without sounding so harsh :) 

"s2n-tls does not officially support 32-bit platforms. 32-bit builds and tests are not run as part of CI, so may or may not work. If you run into an issue with a 32-bit build, you can open an issue and we may or may not be able to help you resolve the problem. However, PRs related to 32-bit builds will likely not be accepted as they cannot be properly tested. We plan to offer more official 32-bit support in the future."